public/app/features/query/components/QueryGroupOptions.tsx

```diff
@@ -358,11 +359,23 @@ export class QueryGroupOptionsEditor extends PureComponent<Props, State> {
         {this.renderQueryCachingTTLOption()}
 
         <div className="gf-form">
-          <InlineFormLabel width={9}>Relative time</InlineFormLabel>
+          <InlineFormLabel
+            width={9}
+            tooltip={
+              <>
+                Overrides the relative time range for individual panels, which causes them to be different than what is
+                selected in the dashboard time picker in the top-right corner of the dashboard. For example to configure
+                the Last 5 minutes the Relative time should be <code>now-5m</code> and <code>5m</code>, or variables
+                like <code>$_relativeTime</code>.
+              </>
+            }
+          >
+            Relative time
+          </InlineFormLabel>
           <Input
             type="text"
             className="width-6"
-            placeholder="1h"
+            placeholder="No override"
```

Comment:
The same comment applies here, `No override` is not a valid sample value for this input.

Reply:
@axelavargas, Thanks, I also agree with this explanation.